File: Permutation-Difference-between-Two-Strings.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var findPermutationDifference = function (s, t) {
    let ans = 0;
    for (let i = 0; i < s.length; i++) {
        let ltr = t.indexOf(s[i]);
        ans += Math.abs(i - ltr);
    }
    return ans;
};

File: Permutation-Difference-between-Two-Strings.cpp
class Solution {
public:
    int findPermutationDifference(string s, string t) {
        int ans = 0;
        for (int i = 0; i < s.size(); i++) {
            int ltr = t.find(s[i]);
            ans += abs(i - ltr);
        }
        return ans;
    }
};